packages/server/src/integration-test/postgres.spec.ts

```diff
@@ -27,7 +27,9 @@ describe("row api - postgres", () => {
   let makeRequest: MakeRequestResponse,
```

Comment:
This test file became really big, with the `beforeEach` actions hard to follow. I will work in a refactor to use better helpers to make it more verbose